sdk/clientcore/annotation-processor-tests/src/main/java/io/clientcore/annotation/processor/test/MockHttpResponse.java

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package io.clientcore.annotation.processor.test;
```

Comment:
nit: artifact id is `tests` and package name is `test`. To be consistent with azure-core-test, we should use `test` everywhere.